data/transaction/apiTransactionResult.go

```diff
@@ -47,36 +47,38 @@ type ApiTransactionResult struct {
 	Status                            TxStatus                  `json:"status,omitempty"`
 }
 
-// SimulationResults is the data transfer object which will hold results for simulation a transaction's execution
-type SimulationResults struct {
-	Status     TxStatus                           `json:"status,omitempty"`
-	FailReason string                             `json:"failReason,omitempty"`
-	ScResults  map[string]*ApiSmartContractResult `json:"scResults,omitempty"`
-	Receipts   map[string]*ApiReceipt             `json:"receipts,omitempty"`
-	Hash       string                             `json:"hash,omitempty"`
-	VMOutput   *vmcommon.VMOutput                 `json:"-"`
-}
+// TODO: move this struct in elrond-go
+//
+//// SimulationResults is the data transfer object which will hold results for simulation a transaction's execution
```

Comment:
this can easily remain here. as part of the data package

Reply:
It has a dependency over `VmOutput` and I wouldn't move that here